pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -369,7 +369,7 @@ func getPhase2Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, netwo
 	return nil, fmt.Errorf("Not implemented")
 }
 
-type BridgePodInterface struct {
+type BridgeBindMechanism struct {
```

Comment:
I think the  whole binding mechanism should be exported to its own package (e.g. `bindmechanism` or `bindmethod`).
And then each concrete struct and method can be implemented in a different file.

Then usage will be: `bind := bindmechanism.Bridge{...}`

Reply:
this makes sense to me, but I prefer to do this in a follow-up PR (certainly on a follow-up commit)